src/routes/events/components/Events.js

```diff
@@ -59,7 +59,7 @@ export default class Events extends React.Component<PropsType> {
           </Page>
         </ScrollView>
       }
-      const error = new ContentNotFoundError({type: 'event', id: event.id, city, language})
+      const error = new ContentNotFoundError({type: 'event', id: path, city, language})
```

Comment:
Maybe ContentNotFoundError should be adjusted as well (id -> path)

Reply:
The id has type `numeric | string`. Therefore it was never indented to be the numeric id we previously used. I think it is fine not to rename it here.